### DISCUSSION TITLE
Welcome to golang-clean-architecture Discussions!

### DISCUSSION THREAD

**Discussion by martinyonatann** [DoWithLogic/golang-clean-architecture#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by newmizanur** [comment]
How do you will compare with https://github.com/bxcodec/go-clean-arch ?

**Reply by martinyonatann** [reply]
outline is almost the same, the difference is only in foldering .. because it uses the same concept and the repository is one of the references for making this repository.
